include/dpp/channel.h

```diff
@@ -157,6 +168,16 @@ struct DPP_EXPORT thread_member
 	 */
 	 thread_member& fill_from_json(nlohmann::json* j);
 };
+/** An object that represents a tag that is able to be applied to a thread in a GUILD_FORUM channel. */
+struct DPP_EXPORT forum_tag : public managed, public json_interface<forum_tag>
+{
+    /** the name of the tag (0-20 characters) */
```

Comment:
these appear to be indented with spaces instead of tabs?

Reply:
Nope, indented with tabs as usual.